tests/components/cardpage.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { expect, describe, it } from "vitest";
+import { render, screen } from "@testing-library/react";
+import { CardPage } from "@/components/cardpage";
+
+describe("<CardPage />", () => {
+  it("should render the CardPage component", () => {
+    render(<CardPage />);
+    expect(screen.getByText("Canada Wildfires")).toBeInTheDocument();
```

Comment:
Adapt this test for when you take out the hard-coded values, maybe inject them as part of the test set up